src/workspace/src/apps.ts

```diff
@@ -1,151 +1,166 @@
-import { App } from '@openfin/workspace'
-import { BASE_URL, VITE_RA_URL, VITE_RT_URL } from './consts'
+import { App } from "@openfin/workspace"
+import { BASE_URL, VITE_RA_URL, VITE_RT_URL } from "./consts"
 
 export async function getApps(): Promise<App[]> {
   return [
-    reactiveTrader,
+    reactiveTraderFx,
+    reactiveTraderCredit,
     reactiveAnalytics,
     limitChecker,
     reactiveWorkspace,
-    liveRatesView,
-    tradesView,
-    analyticsView,
-    reactiveAnalyticsView
+    reactiveTraderFxLiveRatesView,
+    reactiveTraderFxTradesView,
+    reactiveTraderFxAnalyticsView,
+    reactiveAnalyticsView,
   ]
 }
 
-const reactiveTraderManifestUrl = `${VITE_RT_URL}/config/rt-fx.json`
+export const reactiveTraderFx: App = {
+  appId: "reactive-trader-fx",
+  title: "Reactive Trader FX",
+  manifestType: "manifest",
+  description: "Reactive Trader FX",
+  manifest: `${VITE_RT_URL}/config/rt-fx.json`,
+  icons: [
+    {
+      src: `${BASE_URL}/images/icons/reactive-trader.png`,
+    },
+  ],
+  publisher: "Adaptive Financial Consulting",
+  tags: ["FX", "Trading", "Market Data", "Analytics"],
+  images: [{ src: `${BASE_URL}/images/previews/reactive-trader.PNG` }],
+}
 
-export const reactiveTrader: App = {
-  appId: 'reactive-trader',
-  title: 'Reactive Trader',
-  manifestType: 'manifest',
-  description: 'Desktop Reactive Trader',
-  manifest: reactiveTraderManifestUrl,
+export const reactiveTraderCredit: App = {
```

Comment:
just added the full Credit app to start with .. the RT Credit app needs a little work to separate out the RFQ Tiles and then we can add those 3 as Views (see below)